src/edfi-performance-test/edfi_performance_test/factories/descriptors/utils.py

```diff
@@ -24,14 +25,13 @@ def _descriptor_type_to_dict_key(descriptor_type):
     return "{}{}".format(dt[0].lower(), dt[1:])
 
 
-def _descriptor_type_to_endpoint(descriptor_type):
-    return "{}s".format(_descriptor_type_to_dict_key(descriptor_type))
-
-
 def build_descriptor(descriptor_type, value):
-    _namespace = "uri://ed-fi.org/"
     descriptor_type = _normalize_descriptor_type(descriptor_type)
-    return "{}{}#{}".format(_namespace, descriptor_type, value)
+
+    # force first letter to be capitalized
+    descriptor_type = descriptor_type[:1].capitalize() + descriptor_type[1:]
+
+    return f"{'uri://ed-fi.org/'}{descriptor_type}#{value}"
```

Comment:
The case sensitivity on strings is important in Meadowlark, so `XyzDescriptor` and `xyzDescriptor` are two different things.